.github/workflows/build-gluon.yml

```diff
@@ -8,6 +8,19 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
   workflow_dispatch:
+    inputs:
```

Comment:
This misses an option for cases where the site-root is not equal with the root of the repository. In case the site is stored in a subdirectory of the repository, this currently does not work.